HarryJLeigh.FlashCards/FlashCards/Controllers/FlashcardController.cs

```diff
@@ -0,0 +1,68 @@
+using System.Data.SqlClient;
```

Comment:
🟠 **Deprecated**

💡 You should be using `Microsoft.Data.SqlClient`

![image](https://github.com/user-attachments/assets/438ceac8-dbf7-4769-95af-b05051a39f1f)

Reply:
Removed and added Microsoft.Data.SqlClient